cis194/week1/jimfang/Homework01.hs

```diff
@@ -0,0 +1,34 @@
+module Homework01 (
+    toDigits,
+    toDigitsRev,
+    doubleEveryOther,
+    sumDigits,
+    validate
+) where
+
+-- toDigits
+toDigits :: Integer -> [Integer]
+toDigits num
+  | num <= 0 = []
+  | otherwise = toDigits ( num `div` 10 ) ++ [ num `mod` 10 ]
+
+-- toDigitRev
+toDigitsRev :: Integer -> [Integer]
+toDigitRev = reverse . toDigits
```

Comment:
The function name should be `toDigitsRev`.

Reply:
thanks